mycompany-seeq-link-connector-myconnector/build.gradle.kts

```diff
@@ -6,14 +6,17 @@ plugins {
 group = "com.mycompany.seeq.link.connector"
 version = "0.1.0-SNAPSHOT"
 
+project.version = "1.0.0.0"
+val minimumSeeqLinkSdkVersion = "100.0.0.0"
```

Comment:
Hmm. Is it still ok to point to 100.0.0.0? I think we should point to an existing version.